apis/projectcontour/v1/detailedconditions.go

```diff
@@ -113,4 +113,20 @@ type DetailedCondition struct {
 	Warnings []SubCondition `json:"warnings,omitempty"`
 }
 
-const ValidConditionType string = "Valid"
+const (
+	ValidConditionType string = "Valid"
```

Comment:
since these are part of the API, let's add godoc per constant.

Reply:
added